tests/unit/conftest.py

```diff
@@ -6,7 +6,6 @@
 @pytest.fixture()
 def nornir_nautobot_class():
     """Provide True to make tests pass.
-
```

Comment:
To stick with the Google style docs let's keep the space in. Everything looks good.

Reply:
Added space in.